views/newLanding.ejs

```diff
@@ -64,6 +64,7 @@
                         <li><a href="/competitionsIntro">Competitions</a></li>
                         <li><a href="/guest-lectures">Guests</a></li>
                         <li><a href="/comingsoon">Exhibits</a></li>
+                        <li><a href="http://sitp.wissenaire.org/">SITP</a></li>

```

Comment:
Change the "SITP" link text to something more descriptive, like "Summer Internship" or "Internships"